src/__tests__/Header.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import {mount, shallow} from 'enzyme';
+import React from 'react';
+import {Header} from '../Header/Header';
+
+describe('<Header>', () => {
```

Comment:
I see this when I run tests
console.error node_modules/fbjs/lib/warning.js:33
      Warning: Unknown event handler property `onLogout`. It will be ignored.
          in div (created by Header)
          in Header (created by WrapperComponent)
          in WrapperComponent

Reply:
Fixed